### ISSUE TITLE
This is literally scythe anticheat 

### ISSUE THREAD

**Issue by Assassin1065** [FrozenFlameYTT/Oynx-Anti-Cheat#1]
"The code is Legit so perfect I love how much time and effort you put into this code :)" -the idiot FrozenFlameYT. This was not your comment to freely edit and change my message. People like you disgust me. The code from this anticheat was stolen from scythe anticheat.

**Comment by Savageangel7749** [comment]
You didn't even try to replace the full code you can still see there are some scythe commands in it lmao
![Screenshot 2023-02-08 6 23 26 PM](https://user-images.githubusercontent.com/124843236/217675148-065b321e-f34f-480e-9949-a9a3e74a564c.png)

**Comment by Assassin1065** [comment]
Yeah, he just renamed it, didn't even bother to update or fix any bugs.

**Comment by Assassin1065** [comment]
Literally no effort was put into this code. It's just a copy of scythe anticheat with some word replacements to look like you made it yourself. Editing people's comments to make yourself look better will not help you. I was simply gonna state that the code was just stolen, but editing my comments is like spitting in my face.

**Comment by Dream23322** [comment]
>People like you disgust me. The code from this anticheat was stolen from scythe anticheat.

I have my own anticheat called Isolate Anticheat
It is based of scythe anticheat but I am in the process of making it way better, I have already fixed many issues in scythe, so if you come across that, please do not hate. Being a dev is already hard, we want to make it easier. Keep in mind I have not looked at the code on this anticheat and if it is just a rename, that is bad.

**Comment by Dream23322** [comment]
Whats the owner discord

**Comment by Assassin1065** [comment]
@Dream23322 they didn't ask MrDiamond64 for permission to edit his anticheat. On top of that FrozenFlameYT edited people's comments to positively reflect his blatant theft. There's no problem with making an anticheat using his code as long as you ask. But blatantly stealing it and claiming you made 100% of it is a line no one should cross.

**Comment by Assassin1065** [comment]
@Dream23322 it's also just a rename.

**Comment by Assassin1065** [comment]
@Dream23322 https://discord.gg/G3uyJxpk

**Comment by Dream23322** [comment]
@Assassin1065 

Whats the link to this anticheat discord
also check out my anticheat

**Comment by Assassin1065** [comment]
@Dream23322 dm me on discord. xassassin.#0

**Comment by Dream23322** [comment]
ok
im at school tho
